main.cpp

```diff
@@ -146,6 +147,21 @@ void AudioInterface::invertPhase()
     }
 }
 
+int AudioInterface::desireGainLevel(int threshold, int gainLevel)
+ {
+     threshold = 20;
+     int finalLevel = 0;
+     for (int i = 0; i < threshold;  i+= 1)
+     {
+         std::cout << "Initial Gain Level = " << gainLevel << " Gain Level Rised To = " << (i+1) <<std::endl;
+         finalLevel += gainLevel;
+     }
+     
+     std::cout << "Level rised = " << finalLevel << std::endl;
+     
+     return (finalLevel);
+ }
+
```

Comment:
- [ ] use `++i;` 
- [ ] add spaces around the `<<` to improve readability.
- [ ] no need for `( )` around your `return` expressions.